content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -93,25 +100,30 @@ Run your application and you will see traces being exported to the console:
 
 ## Create nested spans
 
-Nested spans let you track work that's nested in nature. For example, the `doWork` function below represents a nested operation. The following sample creates a nested span that tracks the `doWork` function:
+Nested spans let you track work that's nested in nature. For example, the
+`doWork` function below represents a nested operation. The following sample
+creates a nested span that tracks the `doWork` function:
 
 ```javascript
-// Create a span. A span must be closed.
-const parentSpan = tracer.startSpan('main');
-for (let i = 0; i < 10; i += 1) {
-  doWork(parentSpan);
-}
+const mainWork = () => {
+  const parentSpan = tracer.startSpan('main');
+
+  for (let i = 0; i < 3; i += 1) {
+    doWork(parentSpan, i);
+  }
 
-/* ... */
+  // Be sure to end the parent span!
+  parentSpan.end();
+}
 
-function doWork(parent) {
-  // Start another span. In this example, the main function already started a
-  // span, so that'll be the parent span, and this will be a child span.
+function doWork(parent, i) {
+  // To set a child span, we need to mark the parent span as the active span
```

Comment:
Not obvious what "set a child span" means for those not already very familiar with tracing.